mtags/src/main/scala-3/scala/meta/internal/pc/completions/InterpolatorCompletions.scala

```diff
@@ -0,0 +1,251 @@
+package scala.meta.internal.pc.completions
+
+import scala.meta.internal.metals.Fuzzy
+import scala.meta.internal.mtags.MtagsEnrichments.*
+import scala.meta.internal.mtags.MtagsEnrichments.given
+import scala.meta.internal.pc.CompletionFuzzy
+import scala.meta.internal.pc.IndexedContext
+import scala.meta.internal.pc.InterpolationSplice
+
+import dotty.tools.dotc.ast.tpd.*
+import dotty.tools.dotc.core.Constants.Constant
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Names.TermName
+import dotty.tools.dotc.core.Symbols.Symbol
+import dotty.tools.dotc.interactive.Completion
+import dotty.tools.dotc.interactive.Interactive
+import dotty.tools.dotc.util.SourcePosition
+import org.eclipse.{lsp4j as l}
+
+object InterpolatorCompletions:
+
+  def contribute(
+      pos: SourcePosition,
+      completionPos: CompletionPos,
+      indexedContext: IndexedContext,
+      lit: Literal,
+      stringValue: String,
+      path: List[Tree],
+      completions: Completions,
+      snippetsEnabled: Boolean
+  )(using Context) =
+    val text = pos.source.content().mkString
+    InterpolationSplice(pos.span.point, pos.source.content(), text) match
+      case Some(interpolator) =>
+        InterpolatorCompletions.contributeScope(
+          lit,
+          pos,
+          completionPos,
+          interpolator,
+          indexedContext,
+          completions,
+          snippetsEnabled
+        )
+      case None =>
+        InterpolatorCompletions.contributeMember(
+          stringValue,
+          lit,
+          path,
+          text,
+          pos,
+          completions,
+          snippetsEnabled
+        )
+    end match
+  end contribute
+
+  /**
+   * Find the type member if it exits:
+   * For string `s" $Main.metho@@ "` we want to get `metho` string.
+   */
+  private def interpolatorMemberSelect(cursor: Int, s: String): Option[String] =
+    if s.startsWith(".") &&
+      s.length >= 2
+    then
+      if cursor < 0 then None
+      else if s.charAt(1).isUnicodeIdentifierStart then
+        val isValidIdentifier =
+          2.until(cursor).forall(i => s.charAt(i).isUnicodeIdentifierPart)
+        if isValidIdentifier then Some(s.substring(1, cursor))
+        else None
+      else Some("")
+    else None
+  end interpolatorMemberSelect
```

Comment:
Hm. I'm wondering if it might replaced by `CompletionPos.query`. 
What the query is in case of interpolator completions?

Reply:
it's the name after `.` when we don't have a braces

Reply:
Actually `CompletionPos.query` works! Changed!